admin_guide/diagnostics_tool.adoc

```diff
@@ -298,6 +302,24 @@ A user-defined timeout may be set by passing the
 if a newly-created log entry is not able to be queried via Elasticsearch after
 45 seconds.
 
+|`sdn`
+a|This check performs the following cluster-level diagnostics of the
+{product-title} SDN:
+
+* Verify that master hosts can connect to kubelets.
+* Verify that nodes can route packets to each other.
+* Validate node addresses.
+* Validate HostSubnet objects.
+
+If you specify the `*openshift_checks_output_dir*` variable, the check also
+saves networking-related objects from the {product-title} API as well as logs,
+OVS flows, iptables rules, and other network configuration information under the
+specified directory.
+
+This check can help you diagnose pod or infrastructure problems when you cannot
+schedule diagnostic pods or they do not provide enough information to
+troubleshoot the issue.
+
```

Comment:
s/they/the diagnostic pods ? 
Not 100% clear and seems a bit awkward to me.

Reply:
I reworded it a little; is it clearer and less awkward now?

Reply:
@Miciah Maybe: _or the diagnostic pods do not provide enough information_

Reply:
Done.